generator/internal/config/config.go

```diff
@@ -0,0 +1,273 @@
+// Copyright 2024 Google LLC
```

Comment:
nit: it is 2025....

Reply:
Ah, you moved the file, but git thinks it is not a move for whatever reason.  I think 2024 is fine in this case.